generator/apps.py

```diff
@@ -1,6 +1,54 @@
+import os
+
+from urllib.parse import quote, urljoin
+from collections import defaultdict
+from hashlib import md5
+from pathlib import Path
+from typing import Dict
+
 from django.apps import AppConfig
 
 
 class GeneratorConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'generator'
+
+    def __init__(self, app_name, app_module):
+        if bool(int(os.getenv("DEBUG", "0"))):
+            self._monkey_patch_static_node()
+
+        super(GeneratorConfig, self).__init__(app_name, app_module)
+
+    @staticmethod
+    def _monkey_patch_static_node():
```

Comment:
Clever.  I like that we don't have to modify the views/contexts with this approach.